2018 Robot/src/org/usfirst/frc/team4999/robot/RobotMap.java

```diff
@@ -52,11 +55,14 @@
 	
 	public static NetworkTable pidTable = NetworkTableInstance.getDefault().getTable("PID");
 	
-	public static PowerDistributionPanel pdp = new PowerDistributionPanel();
+	//TODO: Debug pdp
+	//public static PowerDistributionPanel pdp = new PowerDistributionPanel();
+
 	
 	public static int liftMotor1PDP = 14;
 	public static int liftMotor2PDP = 15;
 	public static int liftBrakePDP = 4;
```

Comment:
Brake is now a solenoid, not a motor.  See updated Robot system spreadsheet.

Reply:
Yeah, on the list of tasks to do I put update lift code